.travis.yml

```diff
@@ -63,6 +76,7 @@ matrix:
         - JDK='Oracle JDK 8'
         - PYTHON=3.6 PYTHONWARNINGS=ignore
         - RAY_INSTALL_JAVA=1
+        - RAY_ENABLE_MULTI_TENANCY
```

Comment:
We need to add this to bazel config, otherwise, it won't be propagated to the test env var.

Reply:
Here https://github.com/ray-project/ray/blob/71f6f83f1d8fb4816222e8c25cffa74f654631eb/.bazelrc#L103 (We had troubles in GCS actor management because of this before).

Reply:
https://bazel.build/designs/2016/06/21/environment.html

Reply:
You mean I should add `test:ci --test_env=RAY_ENABLE_MULTI_TENANCY` to the bazel config?

Reply:
Yes!